summary/contribution.json

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
the file should be named `summary/openapi.json`. Otherwise, CI will create a duplicate

Reply:
It shouldn't be in there at all, docker should cp on build, but it slipped during local testing